internal/weldr/api.go

```diff
@@ -1928,8 +1928,13 @@ func ostreeResolveRef(location, ref string) (string, error) {
 	return parent, nil
 }
 
-func (api *API) depsolveBlueprintForImageType(bp *blueprint.Blueprint, imageType distro.ImageType) (map[string][]rpmmd.PackageSpec, error) {
-	packageSets := imageType.PackageSets(*bp)
+func (api *API) depsolveBlueprintForImageType(bp blueprint.Blueprint, imageType distro.ImageType) (map[string][]rpmmd.PackageSpec, error) {
+	// Depsolve using the host distro if none has been specified
+	if len(bp.Distro) == 0 {
```

Comment:
I think I usually expect `len()` when working with slices. With strings, I would expect `bp.Distro == ""`, I think it's more idiomatic.